Makefile

```diff
@@ -19,7 +19,10 @@ run-flask:
 
 .PHONY: test
 test: test-requirements
-	./scripts/run_tests.sh
+	flake8 .
+	isort --check-only ./app ./tests
```

Comment:
This removes the `-rc` flag to recursively check directories. 

This behaviour is now default in version 5 of isort HOWEVER, this repo is still using version 4 so we are losing some test coverage here with this change.

https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html#-recursive-or-rc

Either we need to add it back in, or upgrade isort I think

Reply:
Ooh, on the surface: great spot! I changed this based on warnings from another repo.

Then, digging deeper, are you sure you've pulled recently? [It does appear to be on v5](https://github.com/alphagov/document-download-frontend/blob/master/requirements-dev.txt#L4).

Reply:
Oh good spot, I thought I had but I must have not.

In that case, approval coming your way!